public/css/stylesheet.css

```diff
@@ -139,10 +139,54 @@ main {
   height:50px;
   position:relative;
   left:1.1%;
-  font-size:40px
-  margin-top:10;
+  font-size:40px;
 }
 
+.form-box{
```

Comment:
all the form css, are for the enhancement of the forms. As you can see theirs a box surrounding the login and register.